doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -150,3 +149,45 @@ WARNING: You should only import Dashbuilder data to a clean installation of {PRO
 . Click *Import*.
 
 endif::DM[]
+
+ifdef::PAM[]
+
+== Deploying dashboards from {CENTRAL} on Dashbuilder Runtime
+
+You can automatically deploy the dashboards from {CENTRAl} on Dashbuilder Runtime. {CENTRAl} is linked to Dashbuilder Runtime using a gradual export feature.
+
+.Prerequisites
+
+* Dashbuilder Runtime is configured on your system.
+* You have set the `dashbuilder.runtime.multi` system property to `true` in the `standalone.xml` file.
+* You have set the value of `dashbuilder.runtime.location` system property to the Dashbuilder Runtime URL. For example:
++
+[source]
+----
+<property name="dashbuilder.runtime.location" value=" http://localhost:8080"
+----
+* You have set the `dashbuilder.export.dir` system property to the shared directory where Dashbuilder Runtime reads its model. For example:
++
+[source]
+----
+<property name="dashbuilder.export.dir" value="/tmp/dashbuilder/models/"
+----
+
+
+.Procedure
+
+. Start {CENTRAL}, go to *Menu* → *Design* → *Pages*.
+. In the *Components* panel, drag and edit the required component types to the canvas.
+. Click *Save*.
+. Select the *Admin* icon in the top-right corner of the screen and select *Dashbuilder Data Transfer*.
+. On the *Dashbuilder Data Transfer* page, click *Custom export*.
+. Select the pages that you want to include in the ZIP file in the *Export Wizard* panel, and click *Next*.
+. Select *Export Wizard* and click *Open*.
++
+You can see the Dashbuilder Runtime home page. If you are not logged in, you are redirected to the login page.
+. Log in to the Dashbuilder Runtime.
```

Comment:
Seemed like the previous step said you're redirected to the home page and only redirected to the login page if you're not already logged in. So this log-in step here doesn't seem to apply to everyone, only those that need to. So either need to just say it in the previous comment, or leave the previous comment as is (pretty obvious that a user would need to log in), and remove the login step, imo.

Reply:
This makes sense. I have removed the login step.